### ISSUE TITLE
Unable to restart stopped enclave

### ISSUE THREAD

**Issue by barnabasbusa** [kurtosis-tech/kurtosis#853]
### What's your CLI version?

0.80.7

### Description & steps to reproduce

If an an enclave is stopped it cannot be restarted.

Example: 
`kurtosis enclave stop testing`
```shell
kurtosis enclave ls
UUID           Name      Status     Creation Time
872520da8502   testing   STOPPED    Mon, 10 Jul 2023 10:41:35 CEST
```
However, there seem to be no command to start up an enclave e.g: `kurtosis enclave start testing`:
```shell
Manage enclaves
Usage:
  kurtosis enclave [command]

Available Commands:
  add         Creates an enclave
  dump        Dumps information about an enclave to disk
  inspect     Inspect an enclave
  ls          Lists enclaves
  rm          Destroys the specified enclaves
  stop        Stops enclaves

Flags:
  -h, --help   help for enclave

Global Flags:
      --cli-log-level string   Sets the level that the CLI will log at (panic|fatal|error|warning|info|debug|trace) (default "info")

Use "kurtosis enclave [command] --help" for more information about a command.
```

### Desired behavior

Enable a specific enclave to be restarted with `kurtosis enclave start <enclaveName>`
Currently the only way to restart a stopped enclave is by nuking the setup, and start from scratch. Not ideal. 

### What is the severity of this bug?

Papercut; this bug is frustrating, but I have a workaround.

**Comment by mieubrisse** [comment]
Hey @barnabasbusa , thanks for reporting! This is an unfortunate limitation to the way we've currently stored Kurtosis metadata (in memory on the Kurtosis container), and I totally agree - it's very frustrating. We're making progress towards being able to fix this by moving the metadata into a database, but we still have a fair bit to move. So unfortunately can't promise this to be done soon, but we are making progress on it :)

**Comment by mieubrisse** [comment]
We're much, much closer to having this - all the necessary information is now persisted to the APIC's disk. The only missing part now is giving the APIC the ability to start up all the stopped services in its enclave using a startup sequence that respects dependencies & availability checks

**Comment by abhijeetbhagat** [comment]
Yes please! its a bit annoying to not be able to resume a stopped enclave. Starting a new enclave means creating new ports which need to be updated at other places and have contracts deployed again.

**Comment by lyfsn** [comment]
May I ask a question? If I directly use the `docker stop` command to stop all containers started by enclave, and then use `docker start` to restart them, is that possible? What will happen?

**Comment by tedim52** [comment]
@lyfsn  While I've never tried this myself, the APIContainer is responsible for managing services + networking in an enclave, and manipulating state of the enclave directly through docker commands could mess with the APIC's maintained state of the world, making subsequent kurtosis commands a bit weird.
